workers/unity/Packages/com.improbable.gdk.core/Systems/CommandSystem.cs

```diff
@@ -8,6 +8,7 @@ namespace Improbable.Gdk.Core
     [DisableAutoCreation]
     [AlwaysUpdateSystem]
     [UpdateInGroup(typeof(SpatialOSSendGroup.InternalSpatialOSSendGroup))]
+    [UpdateBefore(typeof(ComponentUpdateSystem))]
```

Comment:
Any particular reason for this constraint?

Reply:
There was although this is gone now.